torch/jit/_recursive.py

```diff
@@ -548,6 +580,47 @@ def ignore_overloaded(method_name):
         stubs.append(make_stub_from_method(nn_module, method))
     return overload_stubs + stubs
 
+
+def should_compile_property(cls, property_name):
```

Comment:
Now that we've added ignore, can we just add ignores on the modules themselves instead of maintaining a special list?

Reply:
Yeah good point.

Reply:
Sigh actually we can't do this because you can't dynamically add properties to an instance. You have to add them to the class. :'(